src/researches/passiveVoice/periphrastic/determinePassiveSentencePart.js

```diff
@@ -0,0 +1,35 @@
+const arrayToRegex = require( "../../../stringProcessing/createRegexFromArray.js" );
+const determineSentencePartIsPassive = require( "./determineSentencePartIsPassive.js" );
+
+// Auxiliaries (only needed for German)
+const auxiliariesGerman = require( "../../german/passiveVoice/auxiliaries.js" )().allAuxiliaries;
+
+// Participles
+const getParticiplesGerman = require( "../../german/passiveVoice/getParticiples.js" );
+const getParticiples = require( "./getParticiples.js" );
+
+
+/**
+ * Determines whether a sentence part is passive.
+ *
+ * @param {string} sentencePartText The sentence part to determine voice for.
+ * @param {Array} auxiliaries A list with auxiliaries in this sentence part.
+ * @param {string} language The language of the sentence part.
+
+ * @returns {boolean} Returns true if passive, otherwise returns false.
+ */
+module.exports = function( sentencePartText, auxiliaries, language ) {
+	let participles = [];
+	if ( language === "de" ) {
+		let passive = false;
+		let auxiliaryMatches = sentencePartText.match( arrayToRegex( auxiliariesGerman ) );
```

Comment:
Can the auxiliaries only be loaded when they are used in the condition, instead of when this script is used?

Reply:
same as above

Reply:
> It can't. Browserify cannot deal with dynamic require statements. We wil need to fix this for all word lists, because the bundle size is growing rapidly.